handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1093,8 +1093,6 @@ public final SSLEngineResult unwrap(
     }
 
     private SSLEngineResult sslReadErrorResult(int err, int bytesConsumed, int bytesProduced) throws SSLException {
```

Comment:
future improvement: we don't actually use `err` for anything besides getting the string. We could add a method to netty-tcnative that is `SSL.getLastErrorString()` to save a JNI call. Looks like this is the only place that `getLastErrorNumber` is used in non-unit test code too.

Reply:
👍